File: FreelancerApi1/FreelancerApi1/Models/Recibo.cs
using System;
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;

namespace FreelancerApi1.Models
{
	public class Recibo
	{
		[BsonId]
		[BsonRepresentation(BsonType.ObjectId)]
		public string? _Id { get; set; }

        [BsonElement("titulo")]
        public string titulo { get; set; } = null!;


        public string logo { get; set; } = null!;

		public string moneda { get; set; } = null!;

		public decimal monto { get; set; }



        public string descripcion { get; set; } = null!;

		public string direccion { get; set; } = null!;

		public string nombre { get; set; } = null!;

		public string tipoDocumento { get; set; } = null!;

		public string numeroDocumento { get; set; } = null!;

	}
}

File: FreelancerApi2/modelos/Recibo.js
// const { Schema, SchemaTypeOptions } = require("mongoose")

const {Schema, model} = require("mongoose");


const ReciboSchema = Schema({
    titulo: {
        type: String,
        require: true
    },
    logo: {
        type: String,
        require: true
    },
    moneda: {
        type: String,
        require: true
    },
    monto: {
        type: Number,
        require: true
    },
    descripcion: {
        type: String,
        require: true
    },
    direccion: {
        type: String,
        require: true
    },
    nombre: {
        type: String,
        require: true
    },
    tipoDocumento: {
        type: String,
        require: true
    },
    numeroDocumento: {
        type: String,
        require: true
    }
},{versionKey: false});

module.exports = model("Recibo",ReciboSchema, "recibos");